about.html

```diff
@@ -0,0 +1,118 @@
+              activists who comprise the Creative Commons affiliate network in a different country every two. This year,
+              we
+              hope to expand our invitation list including organizations and invdividuals</p>
+            <footer class="please">
+              Please contact us per Email for any further questions about FREE THINKING SUMMIT GLOBAL
+              2020!<br><strong><u>ftsummit2020@ftkorea.org</u></strong>
+            </footer>
+          </section>
+          <section class="creative">
+            <div class="title-1">FREE THINKING SUMMIT GLOBAL 2020 Logo</div>
+            <div class="horizontal-rule"></div>
+            <h4 class="text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Ab accusamus, nobis eaque harum
+              quidem
+              sunt fugiat. Soluta, omnis facilis?</h4>
+            <img src="images/photos/cc_image.png" alt="Free thinking image and logo" class="cc">
+          </section>
+          <section class="past">
+            <header class="title-2">See the past FT Global Summits</header>
+            <div class="horizontal-rule"></div>
+            <h4 class="description-3">Take a look at the last two FT Global Summits which took place in Buenos Aires and
+              in
+              Warsaw</h4>
+            <ul class="past-one-and-two">
+              <li class="past-one">
+                <div class="year">2018</div>
+                <div class="text">CC Global Summit in Buenos Aires</div>
+              </li>
+              <li class="past-one">
+                <div class="year">2016</div>
+                <div class="text">CC Global Summit in Warsaw</div>
+              </li>
+            </ul>
+          </section>
+          <section class="partner">
+            <div class="title3">Partner</div>
+            <div class="horizontal-rule"></div>
+            <ul class="partners">
+              <li><img src="images/icons/mozilla.svg" alt="mozilla icon" class="partner-element"></li>
+              <li><img src="images/icons/google.svg" alt="google icon" class="partner-element"></li>
+              <li><img src="images/icons/naver.svg" alt="naver icon" class="partner-element"></li>
+              <li><img src="images/icons/daumkakaologo.svg" alt="daumkakaologo icon" class="partner-element"></li>
+              <li><img src="images/icons/airbnb.svg" alt="airbnb icon" class="partner-element"></li>
+            </ul>
+          </section>
+          <footer class="footer-cc">
+            <img src="images/photos/cc_image.png" alt="Free thinking image and logo" class="left cc_image">
+            <h5 class="right">2020 Free Thinking Korea. Some Rights Reserved.</h5>
+            <h5 class="right-desk">2020 Free Thinking Korea. Some Rights Reserved. <br> Lorem, ipsum dolor sit amet consectetur adipisicing elit. Libero praesentium quod, nemo animi eius quidem beatae sequi tenetur saepe neque.</h5>
+          </footer>
```

Comment:
- [ ]  As requested by the previous code reviewer Please also notice, the footer for the about page on the desktop size is different from the one on a mobile display. I think you forget to change it so please make the last footer with dark background and remove partner section from the deskrop version in the about page ✅🔥💯


- Your design
![image](https://user-images.githubusercontent.com/58553711/226077382-44cead6a-5a57-43fb-b694-fcd7ef73be17.png)

- Given
![image](https://user-images.githubusercontent.com/58553711/226077431-a275497f-7bc2-46be-b3b6-c3b0844436ab.png)